### PR TITLE
Improvements to trigger notifications for failed CI tests

### PR DESCRIPTION
**Pull request by Madu86**
The enclosed changes to runtest script and GitHub action files should enable us to see if tests run without failing. Please test and let me know your suggestions.

### CODE REVIEW

File: tools/runtest [hunk 1]
```diff
@@ -776,6 +783,10 @@ else
   done
 fi
 
-exit 0
-
+# shamefully exit if QUICK_CI environment variable is defined and tests are failed.
+if [ ! -z $QUICK_CI ] && [ $ncum_failed_tests -gt 0 ]; then
```
Comment (agoetz) [inline tools/runtest:787]: Why test for `$QUICK_CI`? Why can't we always exit with 0 if successful and 1 if failed. Any shell script should do that. I often check for $? and use this to decide on what to do next.

**Comment by Madu86** [comment]
@agoetz  When running `make test`, make throws a shocking error message for failed tests as if there was a catastrophic error with QUICK executable. :laughter Anyway, I have eliminated  environment variable dependence.

**Comment by vwcruzeiro** [comment]
@Madu86 All tests have passed. I am now going to intentionally break one of them to see if this triggers a failure in CI.

File: test/saved/grad_rpsb_meoh_rhf_sto3g.out.ene [hunk 2]
```diff
@@ -23,6 +23,6 @@
  ELECTRONIC ENERGY    = -2546.045274574
  CORE_CORE REPULSION  =  1688.269305715
  EXT CHARGE REPULSION =    -7.513193312
- TOTAL ENERGY         =  -857.775968859
+ TOTAL ENERGY         =  -957.775968859
```
Comment (Madu86) [inline test/saved/grad_rpsb_meoh_rhf_sto3g.out.ene:26]: @vwcruzeiro This won't work. CIs will run successfully. In runtest script I am comparing .out ref files. Not .out.ene etc.

Comment (vwcruzeiro) [reply test/saved/grad_rpsb_meoh_rhf_sto3g.out.ene:26]: Good catch. Let me try again.

**Comment by Madu86** [comment]
@vwcruzeiro I am done with this and tests also look good. Please go ahead with merging. I want place a second PR.

**Comment by vwcruzeiro** [comment]
Merged!
